### DISCUSSION TITLE
Using object.Save() to insert multiple rows into database table

### DISCUSSION THREAD

**Discussion by cfcoder** [wheels-dev/wheels#1445] [General]
I am finding the following example does not seem to work for me.
```
oNewProjMSXref = model("project_milestones_xref").new(project_id="#oNewProject.id#");

oNewProjMSXref.created    = now();
oNewProjMSXref.created_by = 10;
oNewProjMSXref.updated    = now();
oNewProjMSXref.updated_by = 10;

for(msRow in qInitialMilestoneIds){
	oNewProjMSXref.milestone_id = msRow.milestone_id;
	oNewProjMSXref.save();
}

```
Basically, I change one property and save again but it only saves the first loop. Then it moves on and stops looping, even though there are more rows in the query driving the loop. There should be about 12 new rows saved in the project_milestones_xref table.

I have another similar block of code and it too does not work. It won't even save one row.

Can someone shed light on why?

**Comment by chapmandu** [comment]
You're just saving the first instance. Put the new() method inside the loop.

**Reply by cfcoder** [reply]
Thanks, Adam, that works.
